libs/BLEKit/source/CoreGap.cpp

```diff
@@ -11,8 +11,8 @@ void CoreGap::setDefaultAdvertising()
 
 	_advertising_data_builder.setName(default_advertising_data.name);
 	_advertising_data_builder.setServiceData(
-		leka::service::commands::uuid,	 // TODO: commands::uuid only for compatibility with LekaApp
-		{default_advertising_data.data(), default_advertising_data.size()});
+		leka::service::commands::uuid,	 // ? for compatibility with Leka iOS app BLE implementation
```

Comment:
It is a really a TODO, not an information.

Not well formulated but consider to use something else than the uuid of another device to identify robots.